client/blocks/jetpack-backup-creds-banner/index.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+import getRewindState from 'state/selectors/get-rewind-state';
+import QueryRewindState from 'components/data/query-rewind-state';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug, isJetpackSite } from 'state/sites/selectors';
+
+class JetpackBackupCredsBanner extends Component {
+	render() {
+		const { isJetpack, rewindState, siteId, slug, translate } = this.props;
+
+		return (
+			// TODO: prop for tracks event
+			// TODO: link to settings for creds entry
+			<Fragment>
+				{ siteId && isJetpack && <QueryRewindState siteId={ siteId } /> }
+				{ 'awaitingCredentials' === rewindState.state && (
+					<Banner
+						icon="history"
+						href={
+							rewindState.canAutoconfigure
+								? `/start/rewind-auto-config/?blogid=${ siteId }&siteSlug=${ slug }`
+								: `/settings/security/${ slug }`
+						}
+						title={ translate( 'Add your server credentials' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Enter your server credentials' )` *ES Score: 6*

Reply:
This is a design/wording change.